pkgs/development/python-modules/pylink-square/default.nix

```
@@ -0,0 +1,22 @@
+{ lib, stdenv, buildPythonPackage, python3Packages, fetchPypi }:
```

```suggestion
{ lib, buildPythonPackage, fetchPypi, psutil, six, future }:
```

see ["Python library packages in Nixpkgs"](https://nixos.org/manual/nixpkgs/unstable/#python)